### PR TITLE
Add a link to features using the Stages

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: Stages.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # WHATWG Stages
 
+**[See here](https://github.com/search?q=org%3Awhatwg+type%3Aissue+is%3Aopen+label%3A%22stage%3A+1%22%2C%22stage%3A+2%22%2C%22stage%3A+3%22&type=issues) for the current list of features using the Stages process and their current status.**
```
Comment (annevk) [inline Stages.md:3]: I think we should word this similarly to how we link to anchor permanence issues in /working-mode:

> `See [all features using the Stages process] and their current states.`

I also wonder if we should restrict this to `is:issue`. People might well want to use this on a PR?

Comment (chrishtr) [inline Stages.md:3]:
```suggestion
**See [all features using the Stages process](https://github.com/search?q=org%3Awhatwg+type%3Aissue+is%3Aopen+label%3A%22stage%3A+1%22%2C%22stage%3A+2%22%2C%22stage%3A+3%22&type=issues) and their current states.**
```

File: Stages.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # WHATWG Stages
 
+**See [all features using the Stages process](https://github.com/search?q=org%3Awhatwg+type%3Aissue+is%3Aopen+label%3A%22stage%3A+1%22%2C%22stage%3A+2%22%2C%22stage%3A+3%22&type=issues) and their current states.**
```
Comment (chrishtr) [inline Stages.md:3]:
```suggestion
**See [all features using the Stages process](https://github.com/search?q=org%3Awhatwg+is%3Aopen+label%3A%22stage%3A+1%22%2C%22stage%3A+2%22%2C%22stage%3A+3%22&type=issues) and their current states.**
```

File: Stages.md (hunk 1 above)
Comment (chrishtr) [reply Stages.md]: Done!

**Review by annevk — APPROVED** [review]

**Review by cwilso — APPROVED** [review]

**Comment by cwilso** [comment]
As a minor addition, gonna go and and merge this.

**Pull request merged**